pmd-java/src/main/resources/category/java/errorprone.xml

```diff
@@ -2436,6 +2436,116 @@ public class Foo {
         </example>
     </rule>
 
+    <rule name="ReplaceJavaUtilCalendar"
+          language="java"
+          since="7.16"
+          message="Usage of java.util.Calendar should be replaced with classes from java.time"
```

Comment:
```suggestion
          message="Usage of java.util.Calendar should be replaced with classes from java.time"
          minimumLanguageVersion="1.8"
```

The minimum language version is just a detail, but for completeness.
Since we suggest in the rule description to use the java.time API, the rule should only be executed, if the analyzed project uses Java 8 or later...